### ISSUE TITLE
Logger doesn't handle no output directory.

### ISSUE THREAD

**Issue by tarrow** [ContentMine/getpapers#92]
```
tom@pisces getpapers % bin/getpapers.js -f log.xml -q aardvark -n
info: Saving logs to ./undefined/log.xml
error: No output directory given. You must provide the --outdir argument.
```

**Comment by tarrow** [comment]
Interestingly, we find that the logger writes asynchronously so if there is a log delay before changing to the output directory the log is actually written to the current directory anyway. 

You can see this if you put a 5s setTimeout before changing directory

**Comment by tarrow** [comment]
This is now fixed as of PR #93
